javapoet/src/main/java/com/palantir/javapoet/MethodSpec.java

```diff
@@ -42,7 +42,7 @@
 
 /** A generated constructor or method declaration. */
 public final class MethodSpec {
-    private static final String CONSTRUCTOR = "<init>";
+    public static final String CONSTRUCTOR = "<init>";
```

Comment:
This is an implementation detail of `MethodSpec` and should not be exposed

```suggestion
    private static final String CONSTRUCTOR = "<init>";
```